datavalidation.go

```diff
@@ -155,6 +174,11 @@ func (dv *DataValidation) SetRange(f1, f2 interface{}, t DataValidationType, o D
 		formula2 = fmt.Sprintf("<formula2>%.17g</formula2>", v)
 	case string:
 		formula2 = fmt.Sprintf("<formula2>%s</formula2>", v)
+	case []string:
```

Comment:
When do we need to set `formula2` as a list? This case has no test cases covered.